ocaml/idl/datamodel.ml

```diff
@@ -7714,10 +7737,14 @@ let vm_memory_metrics =
 let vm_vcpu_metrics =
   [
     field ~qualifier:DynamicRO ~ty:Int "number" "Current number of VCPUs" ~persist:true;
-    field ~qualifier:DynamicRO ~ty:(Map (Int, Float)) "utilisation" "Utilisation for all of guest's current VCPUs" ~persist:false;
     field ~qualifier:DynamicRO ~ty:(Map (Int, Int)) "CPU" "VCPU to PCPU map" ~persist:false;
     field ~qualifier:DynamicRO ~ty:(Map (String, String)) "params" "The live equivalent to VM.VCPUs_params" ~persist:false;
     field ~qualifier:DynamicRO ~ty:(Map (Int, Set String)) "flags" "CPU flags (blocked,online,running)" ~persist:false;
+    field ~qualifier:DynamicRO
```

Comment:
It is generally not safe to change to order of the fields, since some API messages are position-based.